Library/Homebrew/cmd/gist-logs.rb

```diff
@@ -1,3 +1,14 @@
+#:  * `gist-logs` [`--new-issue`|`-n`] <formula>
```

Comment:
The cause for the different rendering is not a different Gem version but a very subtle detail in this line: It needs to end with a trailing colon (`:`). Otherwise the whole list will no longer be recognized as a definition list and instead rendered as a plain list (the man page is also rendered quite differently, if you look closely).